Simulator/simulator/visualizer.py

```diff
@@ -137,10 +137,10 @@ class Pillow_Visualizer(Visualizer):
     def __init__(self, env):
         super().__init__(env)
 
-    def get_canopy_image(self, bounds, dir_path, eval, scale=8, identifier="test"):
+    def get_canopy_image(self, bounds, dir_path, eval, scale=8, identifier="test", day=0):
         x_low, y_low, x_high, y_high = bounds
         row_scale, col_scale = (self.env.rows // (x_high - x_low)) * scale, (self.env.cols // (y_high - y_low)) * scale
-        image = Image.new('RGB', (self.env.cols * col_scale, self.env.rows * row_scale), (255, 255, 255))
+        image = Image.new('RGBA', (self.env.cols * col_scale, self.env.rows * row_scale), (255, 255, 255, 0))
```

Comment:
@w07wong are you ok with this?

Reply:
Yeah we don't use visualizer for data_collection if that's what you mean.  If the code works its fine.